docs/paper.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "antimeridian: A Python package for fixing geometries that cross the 180th meridian"
+tags:
+  - Python
+  - geospatial
+  - antimeridian
+authors:
+  - name: Peter Gadomski
+    orcid: 0000-0003-4877-7217
+    corresponding: true
+    affiliation: 1
+  - name: Preston Hartzell
+    orcid: 0000-0002-8293-3706
+    affiliation: 2
+affiliations:
+  - name: Development Seed, USA
+    index: 1
+  - name: Element 84, Inc., USA
+    index: 2
+date: 10 October 2024
+bibliography: paper.bib
+---
+
+## Summary
+
+Locations on and around planet Earth can be represented in a geodetic coordinate system by a latitude, a longitude, and a height.
+Longitude, the "horizontal" dimension, covers the domain from -180° to 180° or 0° and 360°.
+Where the two domain bounds meet is known as the _180th meridian_ or the _antimeridian_.
+
+![Earth map centered on the Pacific ocean, with the 180th meridian highlighted.](./img/antimeridian.jpg)
+
+The GeoJSON specification [@10.17487/RFC7946] describes how antimeridian-crossing shapes should be represented.
+For a variety of reasons, real-world geometries often do not comply with the specification, leading to confusing and unrepresentable geometries.
+Our **antimeridan** package provides Python functions for correcting improper geometries, as well as other related utilities.
+
+## Statement of need
+
+Due to a variety of factors, including the relative lack of populated settlements on the other side of the world, the Prime Meridian (0°) runs through Greenwich, England [@alma992356353405961].
+Before the advent of satellite imagery, relatively few geospatial products crossed the 180th meridian, and so the problem of antimeridian-crossing geometries was usually avoidable.
+The proliferation of satellite remote sensing products, including Earth Observation (EO) imagery, coupled with the ubiquity of interactive online maps, the antimeridian has become a feature that can appear on almost anyone's tablet, web portal, or desktop Geographic Information System (GIS) software.
+There is a a need to create and fix antimeridian-crossing geometries at scale, e.g. for large SpatioTemporal Asset Catalog (STAC) [@STAC_Contributors_SpatioTemporal_Asset_Catalog_2024] catalogs that are used to search and discover petabytes of geospatial data.
+When creating these catalogs, improper antimeridian-crossing geometries need to be corrected before ingesting to a data store to ensure that queries do not break and visualizations do not go haywire.
```

Comment:
Do queryies break? Or just not return info that might exist?

Reply:
They can — the effects can be kind of all over the place, in my experience.